.ci/pipeline_definitions

```diff
@@ -1,14 +1,15 @@
 gardener-extension-registry-cache:
-  template: 'default'
   base_definition:
-    repo: ~
     traits:
       version:
         preprocess: 'inject-commit-hash'
   jobs:
     head-update:
       traits:
-        component_descriptor: ~
+        component_descriptor:
+          ocm_repository: europe-docker.pkg.dev/gardener-project/snapshots
```

Comment:
The registry-cache snapshots are actual built by prow. We use concourse only for releases.
See https://github.com/gardener/ci-infra/blob/8ee30aa71c09d66d0416a18048a6ab85a593cc03/config/jobs/gardener-extension-registry-cache/gardener-extension-registry-cache-test-builds.yaml#L39 and https://github.com/gardener/ci-infra/blob/8ee30aa71c09d66d0416a18048a6ab85a593cc03/config/jobs/gardener-extension-registry-cache/gardener-extension-registry-cache-build-images.yaml#L24.

Reply:
@ialidzhikov : I see. in that case, the pipeline-definition might be adequately adjusted to no longer define component-descriptor trait for head-update-job (I can include this change into this pull-request). wdyt?

Reply:
> in that case, the pipeline-definition might be adequately adjusted to no longer define component-descriptor trait for head-update-job (I can include this change into this pull-request). wdyt?

Sorry, I cannot judge what would this mean. We still need to ability to deploy a revision from the main branch to the dev environment. Would dropping component-descriptor trait for head-update-job still allow this?

Reply:
@ialidzhikov : in case you wish to be able to deploy snapshot-versions w/o triggering release-job, then this is still needed (including the change for head-update's component-descriptor trait).